packages/syft/src/syft/service/worker/worker_image_service.py

```diff
@@ -63,43 +74,97 @@ def submit_dockerfile(
     def build(
         self,
         context: AuthedServiceContext,
-        uid: UID,
+        image: UID,
         tag: str,
-        push: bool = False,
-        container_registry: Optional[str] = None,
+        version: str = "latest",
+        registry: Optional[UID] = None,
     ) -> Union[SyftSuccess, SyftError]:
-        result = self.stash.get_by_uid(credentials=context.credentials, uid=uid)
+        worker_image: SyftWorkerImage = None
+        registry_obj: Optional[SyftImageRegistry] = None
+        image_registry_service: Optional[SyftImageRegistryService] = None
+
+        result = self.stash.get_by_uid(credentials=context.credentials, uid=image)
         if result.is_err():
             return SyftError(
-                message=f"Failed to get image for uid: {uid}. Error: {result.err()}"
+                message=f"Failed to get image for uid: {image}. Error: {result.err()}"
             )
+        worker_image = result.ok()
 
-        worker_image: SyftWorkerImage = result.ok()
```

Comment:
I think `worker_image: SyftWorkerImage = result.ok()` was cleaner. Plus `worker_image: SyftWorkerImage = None` would be a type error, though mypy is broken now.